docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenTracing_SmallRye.adoc

```diff
@@ -0,0 +1,34 @@
+[[MicroProfile_OpenTracing SmallRye]]
+= MicroProfile OpenTracing Subsystem Configuration
+
+Support for https://microprofile.io/project/eclipse/microprofile-opentracing[Eclipse MicroProfile OpenTracing] is provided by
+ the _microprofile-opentracing-smallrye_ subsystem.
+
+[[required-extension]]
+== Required Extension
+
+This extension is included in the standard configurations included in the WildFly distribution.
+
+You can also add the extension to a configuration without it either by adding
+an `<extension module="org.wildfly.extension.microprofile.opentracing-smallrye"/>`
+element to the xml or by using the following CLI operation:
+
+[source,ruby]
+----
+[standalone@localhost:9990 /]/extension=org.wildfly.extension.microprofile.opentracing-smallrye:add
+----
```

Comment:
You also need to add the subsystem for the functionality to take effect

Reply:
cc @jmesnil , as it probably applies to his MP-Config as well:

https://github.com/jmesnil/wildfly/blob/babea3b0f925fab044a3e9a04bfe30788f9b58c5/docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Config_SmallRye.adoc